components/desktop/xscreensaver/Makefile

```diff
@@ -70,28 +72,34 @@ CONFIGURE_OPTIONS+=	--with-configdir=/usr/lib/xscreensaver/config/control-center
 CONFIGURE_OPTIONS+=	--with-image-directory=/usr/share/pixmaps/backgrounds
 CONFIGURE_OPTIONS+=	--with-text-file=/etc/motd
 CONFIGURE_OPTIONS+=	--with-x-app-defaults=/usr/share/X11/app-defaults
+CONFIGURE_OPTIONS+=	--with-jpeg
+CONFIGURE_OPTIONS+=	--with-png
+CONFIGURE_OPTIONS+=	--with-gl
 
 COMPONENT_INSTALL_ENV += MAKEFLAGS=-e install_prefix=$(PROTO_DIR)
 
 ../rss-glx/build/i86/.built:
 	(cd ../rss-glx ; $(GMAKE) build)
 
 # common targets
-build:		$(BUILD_32)
+build:		$(BUILD_64)
 
-install:	$(INSTALL_32) ../rss-glx/build/i86/.built
+install:	$(INSTALL_64) ../rss-glx/build/i86/.built
 	perl ../rss-glx/build/i86/utils/rss-glx_install.pl $(PROTO_DIR)/usr/share/X11/app-defaults/XScreenSaver
 test:		$(NO_TESTS)
 
 REQUIRED_PACKAGES += gnome/config/gconf
+REQUIRED_PACKAGES += image/library/libjpeg
+REQUIRED_PACKAGES += image/library/libpng16
+REQUIRED_PACKAGES += library/c++/harfbuzz
 REQUIRED_PACKAGES += library/desktop/atk
 REQUIRED_PACKAGES += library/desktop/gdk-pixbuf
 REQUIRED_PACKAGES += library/desktop/gtk2
 REQUIRED_PACKAGES += library/desktop/pango
 REQUIRED_PACKAGES += library/glib2
 REQUIRED_PACKAGES += library/libxml2
 REQUIRED_PACKAGES += library/perl-5/xml-parser
-REQUIRED_PACKAGES += runtime/perl-522
```

Comment:
Alexander suggested that both Perl versions should be kept. I am not sure it sill applies.

Reply:
OK, I kept Perl 5.22.